### PR TITLE
fix issues/#116

### PR DESCRIPTION
**Pull request by shine2lay**
also added a test case to check it

---
This change is [Reviewable](https://reviewable.io/reviews/wetrustplatform/rosca-contracts/117)

### CODE REVIEW

File: test/endOfROSCAUnitTest.js [hunk 1]
```diff
@@ -36,6 +36,23 @@ contract('end of ROSCA unit test', function(accounts) {
       }
     }
 
+	  it("throws if there are no more fees to be collected",
```
Comment (rmerom) [inline test/endOfROSCAUnitTest.js]: nit: the test suite currently stars with the "normal" flow, then checks for the edge cases. We might want to keep that, by moving the new test past the "normal-flow" ones,

**Review by rmerom — APPROVED** [review]

**Comment by shine2lay** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[test/endOfROSCAUnitTest.js, line 39 at r1](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/117#-KcU8kXydzsBzXJdoUtP-r1-39:-KcUj8RoqcEgClGRiNjQ:b-896fix) ([raw file](https://github.com/wetrustplatform/rosca-contracts/blob/787e1c11ad1c9cdd8cfdae3aa04c7a8609327ce4/test/endOfROSCAUnitTest.js#L39)):*
<details><summary><i>Previously, rmerom (Ron Merom) wrote…</i></summary><blockquote>

nit: the test suite currently stars with the "normal" flow, then checks for the edge cases. We might want to keep that, by moving the new test past the "normal-flow" ones,
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wetrustplatform/rosca-contracts/117)*
